### ISSUE TITLE
Problem with F1 Metric in Fine Tuning

### ISSUE THREAD

**Issue by harkous** [bfelbo/DeepMoji#5]
Hi,

Thanks for the nice and useful repository!

Currently, using the F1 metric instead of the accuracy does not work when fine tuning. For that to work, the global variables might have to be changed at this line ('weighted'->'weighted_f1'):
https://github.com/bfelbo/DeepMoji/blob/2ad102ada961faa82f470814268f0bdef54c3a56/deepmoji/global_variables.py#L27

Also, the fine-tuning code has to be modified at this line as there is no 'weighted_f1' average option for `f1_score`:
https://github.com/bfelbo/DeepMoji/blob/2ad102ada961faa82f470814268f0bdef54c3a56/deepmoji/finetuning.py#L471

**Comment by bfelbo** [comment]
Ah, I recently renamed 'weighted' to 'weighted_f1' to make it more easily understandable, but must have done it a bit too fast. Thanks for reporting these issues! I've pushed a fix now.
